site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -69,7 +66,7 @@ The following is an example of the permissions part of a [manifest][doc-manifest
 
 ## Host permissions {: #host-permissions }
```

Comment:
The aside warning developers that host permissions trigger a warning was placed on purpose under the host permissions section. Based on developer feedback, many beginner developers are not aware that host permissions trigger a warning, including those declared under content scripts in the manifest. I want to leave the warning in the host permission section in case developers do not continue scrolling down to the Permission Warning section.

Reply:
What about the use case where developers go directly to the host permissions section from a direct link without starting to read at the very top? Just curious 🤔

Reply:
I apologize for the wording of that. It was harsher than it should have been.

That scenario has exactly the problem I just described. If someone clicks from the top, they have the opportunity to learn about the warnings, but they're probably missing them because they see the link and click through. Readers coming from outside don't even have the opportunity to see that information. 

Another example is [this aside](https://developer.chrome.com/docs/extensions/mv3/declare_permissions/#:~:text=with%20warnings.-,Consider%20using,-optional%20permissions%20wherever) about optional permissions. It links to chrome.permissions, which doesn't appear to even mention what's in the aside. Someone who lands on the chrome.permissions page will never get this information.

I think of asides as for when readers miss things. I'm suggesting that the problem here is that we missed things.

(If we had content for optional host permissions, I'd say make the top a bulleted list and get rid of the definitions in favor of having everything for each manifest key in one place. But this is just a cleanup pass. Maybe later.)

Reply:
I'll open a PR to show you what I have in mind after this one is merged. We can leave the aside in for now.